pkg/filesystem/local_directory.go

```diff
@@ -354,6 +355,24 @@ func (d *localDirectory) Symlink(oldName string, newName string) error {
 	return unix.Symlinkat(oldName, d.fd, newName)
 }
 
+func (d *localDirectory) Chtimes(name string, atime, mtime time.Time) error {
+	if err := validateFilename(name); err != nil {
+		return err
+	}
+	defer runtime.KeepAlive(d)
+
+	var ts [2]unix.Timespec
+	var err error
+	if ts[0], err = unix.TimeToTimespec(atime); err != nil {
+		return err
```

Comment:
Could you add `util.StatusWrapWithCode(err, codes.InvalidArgument, "Cannot convert access time")` here?